### PR TITLE
Adds Voter registration tool

### PR DESCRIPTION
**Pull request by benhammondmusic**
# Description and Motivation

- Adds new announcement banner to Explore the Data
- Button opens new modal showing the Vote.org iframe
- e2e test rewritten to be faster and to handle modal behavior

## todo

- [x] can banner title be the link? Keep it as small as possible? If not; more distinct text between title and link
- [x] add quote from rep. Lewis re: voting ?
- [x] clean up vertical modal around widget 
- [x] move X to close modal up to top right 

## Has this been tested? How?

- passing

## Types of changes

(leave all that apply)

- New content or feature

### CODE REVIEW

**Comment by benhammondmusic** [comment]
Initial structural ideas
Screenshot 2024-07-09 at 11 54 01 AM
Screenshot 2024-07-09 at 11 54 06 AM

**Review by kccrtv — APPROVED** [review]
👍🏼

File: frontend/src/reports/ui/VoteDotOrgBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import HowToVoteIcon from '@mui/icons-material/HowToVote'
+import HetNotice from '../../styles/HetComponents/HetNotice'
+import type { DataTypeId } from '../../data/config/MetricConfig'
+import { useParamState } from '../../utils/hooks/useParamState'
+import { VOTE_DOT_ORG_PARAM_KEY } from '../../utils/urlutils'
+import HetLinkButton from '../../styles/HetComponents/HetLinkButton'
+
+export const LIFELINE_IDS: DataTypeId[] = ['suicide', 'gun_violence_suicide']
```
Comment (eriwarr) [inline frontend/src/reports/ui/VoteDotOrgBanner.tsx:8]: This is also defined in `LifelineAlert.tsx`. Is it needed here?

**Review by eriwarr — APPROVED** [review]
Looks great!

File: frontend/src/reports/ui/VoteDotOrgBanner.tsx (hunk 1 above)
Comment (benhammondmusic) [reply frontend/src/reports/ui/VoteDotOrgBanner.tsx]: nice catch!

**Pull request merged**